src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,99 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name="users")
+public class User {
+
+    public User(){
+
+    }
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private String userName;
+
+    @Column(nullable = false)
+    private LocalDate birthDate;
+
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
You don't want CascadeType.ALL, because a Book can exist independantly of a User. What cascade types do you think you need here? The only thing you want is to keep the JoinTable updated. Let me know if you don't find an anwer.